src/pages/case-study/central-working-changing-how-you-work.js

```diff
@@ -0,0 +1,385 @@
+import React, { Fragment } from 'react'
+import { graphql } from 'gatsby'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import ReactMarkdown from 'react-markdown'
+import { generate } from 'shortid'
+
+import { Grid, Col } from '../../components/grid'
+import GreyBackground from '../../components/Common/GreyBackground'
+import BlackBackground from '../../components/Common/BlackBackground'
+import TanBackground from '../../components/Common/TanBackground'
+
+import Layout from '../../components/layout'
+import Head from '../../components/Common/Head'
+import Hr from '../../components/Common/Hr'
+import CaseStudyHero from '../../components/Common/CaseStudyCards/CaseStudyHero'
+import { SectionTitle, BodyPrimary } from '../../components/Typography'
+import {
+  TextColumnsBlock,
+  shouldRender,
+  normalise,
+  normaliseAll,
+  BlockRow
+} from '../../components/Common/CaseStudyCards/CaseStudyBlocks'
+
+import CaseStudyPreview from '../../components/Common/CaseStudyCards/CaseStudyPreview'
+import Image from '../../components/Common/Image'
+
+const GreyBodyPrimary = styled(BodyPrimary)`
+  color: ${({ theme }) => theme.colors.secondaryText};
+`
+
+const Block3Col = styled(Col)`
+  padding-top: ${({ theme }) => theme.space[3]};
+`
+
+const Block4ImageCol = styled(Col)`
+  padding-top: ${({ theme }) => theme.space[4]};
+`
+
+const Block6ImageCol = styled(Col)`
+  padding-top: ${({ theme }) => theme.space[4]};
+`
+
+const Block7ImageCol = styled(Col)`
+  ${breakpoint('smallPhone', 'phone')`
+    padding-top: ${({ theme }) => theme.space[4]};
+  `}
+`
+
+const Block9ImageCol = styled(Col)`
+  ${breakpoint('smallPhone')`
+      padding-top: ${({ theme }) => theme.space[4]}
+  `}
+`
+
+const StyledHr = styled(Hr)`
+  margin-left: auto;
+  margin-right: auto;
+
```

Comment:
Does this work with `margin: 0 auto`; ? Nicer if it does. Dont know if some margins top bottom are set on Hr

Reply:
Hr has margin top and padding bottom, this is why I've set it here explicitly

Reply:
Alright. Makes sense then. I was too lazy to check 😆